packages/suite-data/files/translations/es.json

```diff
@@ -0,0 +1,1489 @@
+	"TR_WIPE_DEVICE_CHECKBOX_2_TITLE": "Entiendo que esta acción no afecta a mis fondos",
+	"TR_WIPE_DEVICE_HEADING": "Antes de borrar tu dispositivo…",
+	"TR_WIPE_DEVICE_TEXT": "Al borrar el dispositivo se elimina todo su contenido. Borra el dispositivo sólo si tienes la semilla de recuperación contigo o cuando no haya fondos en el dispositivo.",
+	"TR_WIPING_YOUR_DEVICE": "Borrar el dispositivo elimina toda su información. Borra tu dispositivo sólo si tienes tu semilla de recuperación a mano o si no hay fondos almacenados en este dispositivo.",
+	"TR_WORDSNumber of words. For example: 12 words": "{count} palabras",
+	"TR_WORD_DOES_NOT_EXISTIn recovery or dry run, appears when user types a string that is not a substring of any word included in bip39 word list.": "La palabra \"{word}\" no existe en la lista de palabras bip39.",
+	"TR_WRONG_PIN_ENTERED": "Has introducido un PIN incorrecto",
+	"TR_WRONG_PIN_ENTERED_DESCRIPTION": "¿Has notado que la matriz de tu dispositivo ha cambiado?",
+	"TR_XPUB_MODAL_CLIPBOARD": "Copiar clave pública",
+	"TR_XPUB_MODAL_TITLE": "{networkName} Cuenta {accountIndex} clave pública (XPUB)",
+	"TR_XPUB_MODAL_TITLE_METADATAaccountLabel is user defined name of account, might be pretty much anything.": "{accountLabel} clave pública (XPUB)",
+	"TR_XRP_RESERVE_INFO": "Las direcciones Ripple requieren un saldo mínimo de {minBalance} XRP para activar y mantener la cuenta.",
+	"TR_YOUR_CURRENT_FIRMWARE": "Tu versión actual de firmware es {version}.",
+	"TR_YOUR_CURRENT_VERSION": "Actualmente está ejecutando la versión {version}.",
+	"TR_YOUR_DEVICE_IS_CONNECTED_BUT_UNREADABLE": "Tu dispositivo está conectado correctamente, pero tu navegador de Internet no puede comunicarse con él en este momento. Necesitarás instalar Trezor Bridge.",
+	"TR_YOUR_DEVICE_IS_SEEDLESS": "Tu dispositivo está en modo sin semilla y no está permitido utilizarlo con esta wallet.",
+	"TR_YOUR_LABELING_IS_SYNCED": "Tu etiquetado está sincronizado con el proveedor de almacenamiento en la nube. Tus datos son seguros, solo tu Trezor puede descifrarlos.",
+	"TR_YOUR_LABELING_IS_SYNCED_LOCALLY": "Las etiquetas se guardan localmente en tu máquina.",
+	"TR_YOUR_NEW_VERSION": "La versión {version} está disponible.",
+	"TR_YOUR_TREZOR_IS_ALMOST_READY": "Tu Trezor está casi listo",
+	"TR_YOUR_TREZOR_IS_NOT_BACKED_UP": "Tu Trezor no está respaldado.",
+	"TR_YOUR_WALLET_IS_READY_WHAT": "Tu Wallet está lista. ¿Qué hacer ahora?",
+	"TR_YOU_CAN_SELECT_EITHER": "Puedes seleccionar un nuevo Trezor o cualquier otro dispositivo Trezor que haya sido utilizado e inicializado antes.",
+	"TR_YOU_EITHER_HAVE_T1": "Tienes una semilla que contiene 12, 18 o 24 palabras.",
+	"TR_YOU_EITHER_HAVE_T2": "Tienes una semilla que contiene 12, 18, 20, 24 o 33 palabras.",
+	"TR_YOU_MAY_CHOSE_EITHER_STANDARD": "You might chose either standard backup type or Shamir backup",
+	"TR_YOU_WERE_DISCONNECTED_DOT": "Has sido desconectado.",
+	"TX_CONFIRMATIONS_EXPLAIN": "TODO TODO TODO ExPlAnAtIoN"
```

Comment:
no es finito hombres karamba

Reply:
@FrankyFFV :smile:

Reply:
otherwise excellent work :+1:

Reply:
I don’t understand your comment 😂

Reply:
TX_CONFIRMATIONS_EXPLAIN is not used anywhere, so it's not a problem.